lib/systems/parse.nix

```diff
@@ -280,6 +280,7 @@ rec {
     windows = { execFormat = pe;      families = { }; };
     ghcjs   = { execFormat = unknown; families = { }; };
     genode  = { execFormat = elf;     families = { }; };
+    unknown = { execFormat = unknown; families = { }; };
```

Comment:
I am a bit uncomfortable adding `unknown` as it's not actually an OS (`kernel` should be renamed `OS`). But yes there doesn't seem  to be many good options, maybe we could even temporarily call it `webghc`, just like the `ghcjs` up above. 

I would like to use to use `browser`, cause at the end of the day regardless of what sort of libc you get there is the externally implemented wasm interfaces that seem to be the moral equivalent of the syscalls---interfaces into the hosting system (OS) which is outsides the program's own control.

Reply:
I did some study of this, but did not find very convincing solution to what the OS should be called.

Regarding the `unknown` OS though, it has been widely used in the wasm cross-compilation target.
In fact it is present in the code mentioned at the top of this file http://llvm.org/doxygen/Triple_8cpp_source.html
`case UnknownOS: return "unknown";`
(Also there is no `none` option).

Nevertheless  at this stage getting this correct is not even that important. If `wasm32-unknown-none-musl` is the one which works without adding `unknown` to the OS list, I am ok with it. Only when the changes are being upstreamed to ghc this will have to be fixed properly.